app/views/layouts/_foot.html.erb

```diff
@@ -43,3 +43,22 @@
   check();
 }).call(this);
 </script>
+
+<div id="bz-tz-warning" style="display: none;">
+  Please ensure your <a href="/profile/settings">user settings</a> shows the right time zone!
```

Comment:
Maybe "We noticed that your current timezone doesn't match your <a href="/profile/settings">settings</a>.  Consider changing your <a href="/profile/settings">settings</a> so your due dates show the right time!"